src/plugins/elasticsearch.js

```diff
@@ -21,9 +21,15 @@ function createWrapRequest (tracer, config) {
         }
       })
 
-      if (JSON.stringify(params.body)) {
-        span.setTag('elasticsearch.body', JSON.stringify(params.body))
+      const indexPath = JSON.stringify(params.path).split('/')[1]
+
+      if (indexPath.substring(0, 1) !== '_') {
+        span.setTag('elasticsearch.index', JSON.stringify(params.path).split('/')[1])
+      }
+      if (JSON.stringify(params.body) && JSON.stringify(params.body.query)) {
```

Comment:
These can be nontrivial operations.  Why not check existence of body and body.query before attempting to convert the query?  Also, what if the attempt throws a TypeError?